docs/README.md

```diff
@@ -0,0 +1,349 @@
+
+<h1 id="devcycle-test-harness-proxy">DevCycle Test Harness - Proxy v1.0.0</h1>
+
+# Overview
+The SDK proxy server is responsible for wrapping the SDK under test with an HTTP interface that can be used to call any of its public-facing methods with an arbitrary set of arguments,
+and returning the response to the caller (which will be the harness server). It will also allow configuration options to be passed which override the base API URL the SDK makes requests to.
+Setting that URL to the test harness server’s address allows the harness to mock the backend APIs to return data back (or to trigger errors).
+
+### About `entityType`
+To check that the SDK methods are returning a correct type of an instance, they will all contain an `entityType` in their response. `enityType` can be one of five values:
+  `Variable`, `User`, `Feature`, `Object`, `Void`
+
+<h1 id="devcycle-test-harness-proxy-client">Client</h1>
+
+SDK Client
+
+<a id="opIdcreateClient"></a>
+
+`POST /client`
+
+*Create a DVC client instance*
+
+> Body parameter
+
+```json
+{
+  "clientId": "string",
+  "sdkKey": "string",
+  "options": {},
+  "enableCloudBucketing": true
+}
+```
+
+<h3 id="createclient-parameters">Parameters</h3>
+
+|Name|In|Type|Required|Description|
+|---|---|---|---|---|
+|`clientId`|body|string|true|A unique ID that will be used to reference the client once it has been created|
+|`sdkKey`|body|string|false|A unique key used to initialize the DVC SDK|
+|`options`|body|object|false|An object of launch parameters. See [DevCycle Docs](https://docs.devcycle.com/docs/sdk/server-side-sdks/node#initialization-options) for a full list of options|
+|`enableCloudBucketing`|body|boolean|false|An option to use the cloud bucketing version of the SDK|
+
+> Example responses
+
+> Created
+
+```json
+{
+  "message": "Success"
+}
+```
+
+```json
```

Comment:
in the error case (where the SDK threw an error) is the status code of the response the same and its just the message that tells us about the error? Should this be "exception" instead?

Reply:
yeah the status code should be 200 but the body should contain `exception` instead. I'll make the change